modules/ROOT/pages/_partials/caveats.adoc

```diff
@@ -0,0 +1,7 @@
+
+// Local ASN
+// tag::localAsnNote[]
+Although the local ASN isn't used for static routing, 
```

Comment:
This is relevant when creating the private space's first VPN, if it's using static routing. But it's currently showing under the "Dynamic (BGP)" section in the doc.

Reply:
Fixed, thanks